src/types/lib.ts

```diff
@@ -24,7 +37,7 @@ export type CallDetails = {
 
 export type Invocation = CallDetails & { signature?: Signature };
 
-export type Call = CallDetails & { entrypoint: string };
+export type Call = CallDetails & { entrypoint: BigNumberish };
```

Comment:
entrypoint should only be string and not bignumberish, as the property will get processed using `getSelectorFromName`, which expects human readable selectors